autoload/neomake.vim

```diff
@@ -577,6 +577,7 @@ function! neomake#MakeHandler(job_id, data, event_type) abort
 
         " Show the current line's error
         call neomake#CursorMoved()
```

Comment:
Yes, we can remove this one. The next one should stay, otherwise if there is a new error it won't be displayed (because the line number isn't changed).